tox.ini

```diff
@@ -17,6 +19,9 @@ commands =
     python scripts/print_env.py
 
 [testenv]
+# N.B.: We need modern setuptools downloaded OOB by virtualenv to work with the Python 3.12 case.
```

Comment:
Nit so the comment is more accessible:

```suggestion
# N.B.: We need modern setuptools downloaded out-of-bound by virtualenv to work with the Python 3.12 case.
```

Reply:
I guess you proved your point. It's out of band.

Reply:
Fixed.